pkg/kwokctl/cmd/delete/cluster/cluster.go

```diff
@@ -52,62 +53,92 @@ func NewCommand(ctx context.Context) *cobra.Command {
 		},
 	}
 	cmd.Flags().StringVar(&flags.Kubeconfig, "kubeconfig", flags.Kubeconfig, "The path to the kubeconfig file that will remove the deleted cluster")
+	cmd.Flags().BoolVar(&flags.All, "all", flags.All, "Delete all clusters")
+
 	return cmd
 }
 
 func runE(ctx context.Context, flags *flagpole) error {
-	name := config.ClusterName(flags.Name)
-	workdir := path.Join(config.ClustersDir, flags.Name)
-
-	logger := log.FromContext(ctx)
-	logger = logger.With("cluster", flags.Name)
-	ctx = log.NewContext(ctx, logger)
+    if flags.All {
+        return deleteAllClusters(ctx, flags)
+    }
 
-	var err error
-	flags.Kubeconfig, err = path.Expand(flags.Kubeconfig)
-	if err != nil {
-		return err
-	}
-
-	rt, err := runtime.DefaultRegistry.Load(ctx, name, workdir)
-	if err != nil {
-		if errors.Is(err, os.ErrNotExist) {
-			logger.Warn("Cluster does not exist")
-		}
-		return err
-	}
+    return deleteCluster(ctx, flags.Name, flags.Kubeconfig)
+}
 
-	// Stop the cluster
-	start := time.Now()
-	logger.Info("Cluster is stopping")
-	err = rt.Down(ctx)
-	if err != nil {
-		return err
-	}
-	logger.Info("Cluster is stopped",
-		"elapsed", time.Since(start),
-	)
-
-	// Delete the cluster
-	start = time.Now()
-	logger.Info("Cluster is deleting")
-	if flags.Kubeconfig != "" {
-		err = rt.RemoveContext(ctx, flags.Kubeconfig)
-		if err != nil {
-			logger.Error("Failed to remove context from kubeconfig", err,
-				"kubeconfig", flags.Kubeconfig,
-			)
-		}
-		logger.Debug("Remove context from kubeconfig",
-			"kubeconfig", flags.Kubeconfig,
-		)
-	}
-	err = rt.Uninstall(ctx)
-	if err != nil {
-		return err
-	}
-	logger.Info("Cluster is deleted",
-		"elapsed", time.Since(start),
-	)
-	return nil
+func deleteAllClusters(ctx context.Context, flags *flagpole) error {
+    clustersDir := config.ClustersDir
+    clusters, err := os.ReadDir(clustersDir)
+    if err != nil {
+        return err
+    }
```

Comment:
```suggestion
    clusters, err := runtime.ListClusters(ctx)
	if err != nil {
		return err
	}
```

Reply:
<img width="348" alt="image" src="https://github.com/kubernetes-sigs/kwok/assets/6565744/e0dbea24-240c-49ae-a1ba-0f24eb1386b6">

The cluster names are different